src/components/ItemList.svelte

```diff
@@ -149,7 +151,9 @@
                       />
                     {/each}
                   {/if}
-                  {#if isExpired(item.expires)}
+                  {#if item.in_source === false}
```

Comment:
```suggestion
                  {#if !item.in_source}
```

Reply:
i'm wondering if it's possible for `item.in_source` to ever be undefined or null?

Reply:
It shouldn't be, no. This code is a holdover from an earlier approach that I abandoned.